SofaKernel/modules/SofaExplicitOdeSolver/EulerSolver.cpp

```diff
@@ -46,92 +46,132 @@ using namespace sofa::defaulttype;
 using namespace sofa::helper;
 using namespace core::behavior;
 
-int EulerSolverClass = core::RegisterObject("A simple explicit time integrator")
-        .add< EulerSolver >()
+int EulerExplicitSolverClass = core::RegisterObject("A simple explicit time integrator")
```

Comment:
Add deprecation message for the next release about using aliases

Reply:
and add a static int EulerExplicitSolverClass to remove a warning.